src/main/kotlin/platform/TUL.kt

```diff
@@ -28,7 +28,8 @@ object TUL {
                                 tupper.name,
                                 tupper.nick,
                                 null,
-                                null
+                                null,
+                                arrayListOf()
```

Comment:
Tupperbox members have a `brackets` field, which is a 2 element array containing a prefix and a suffix

Reply:
will do!

Reply:
Added support and confirmed working! thanks for the spec info.

![javaw_RjHuU64yX8](https://user-images.githubusercontent.com/55819817/233751748-ac97307e-e5c1-4197-b8af-2fd3a7495256.png)